### PR TITLE
Added option to ignore CRC validation

### PR DESCRIPTION
**Pull request by egourlao**
Until the account server implements CRC settings, the packet forwarder currently sends upstream CRC-valid packets and no-CRC packets, and doesn't send upstream packets with an invalid CRC. The packet forwarder should, if the user wants it, ignore CRC validation and send every RX packet upstream.

This PR, following issue #32 , implements an `--ignore-crc` flag. When this flag is used at startup, packets are sent upstream regardless of CRC validation.

### CODE REVIEW

File: cmd/start.go [hunk 1]
```diff
@@ -59,6 +59,11 @@ var startCmd = &cobra.Command{
 			}
 		}
 
+		crcCheck := !config.GetBool("ignore-crc")
```
Comment (htdvisser) [inline cmd/start.go]: `crcCheck` should be consistent with config: use `ignoreCRC` or have the `crcCheck` disabled with `--crc-check=false`

File: cmd/start.go [hunk 2]
```diff
@@ -59,6 +59,11 @@ var startCmd = &cobra.Command{
...
+		if !crcCheck {
+			ctx.Warn("CRC check disabled, non-CRC valid packets will be sent upstream.")
```
Comment (htdvisser) [inline cmd/start.go]: "non-CRC valid packets"... sorry, what?

**Review by htdvisser — APPROVED** [review]
This could use some naming/wording improvements, but the logic is ok

File: pktfwd/manager.go [hunk 3]
```diff
@@ -35,6 +35,7 @@ type Manager struct {
 	bootTimeSetters     multipleBootTimeSetter
 	foundBootTime       bool
 	isGPS               bool
+	ignoreCrc           bool
```
Comment (johanstokking) [inline pktfwd/manager.go]: `ignoreCRC`

File: pktfwd/network.go [hunk 4]
```diff
@@ -30,6 +30,7 @@ type TTNConfig struct {
 	Version             string
 	GatewayDescription  string
 	DownlinksSendMargin time.Duration
+	IgnoreCrc           bool
```
Comment (johanstokking) [inline pktfwd/network.go]: `IgnoreCRC`

**Review by johanstokking — CHANGES_REQUESTED** [review]

**Review by johanstokking — APPROVED** [review]

**Pull request merged**